lib/charms/opensearch/v0/opensearch_backups.py

```diff
@@ -160,9 +161,30 @@ def __init__(self, charm: Object):
             self.charm.on.check_restore_status_action, self._on_check_restore_status_action
         )
 
+    @property
+    def type(self) -> str:
```

Comment:
we should probably make this and all the following properties protected (name starts with `_`)